api/docs/http_file_metadata.md

```diff
@@ -12,7 +12,7 @@ the following three types:
 * symbolic link
 
 Note that an `:environment` must be specified in the endpoint, but is actually ignored since the puppet file server
-is not environment-specific.  (In fact, the specified `:environment` does even need to be valid.)
+is not environment-specific.
```

Comment:
Hey, is this actually true? Because if you're asking for anything under `/file_metadata/<ENVIRONMENT>/modules/`, it should matter a lot, and I believe I wrote docs assuming it does.